src/views/CardStack/CardStack.js

```diff
@@ -294,7 +298,7 @@ class CardStack extends React.Component<Props> {
           ? layout.height.__getValue()
           : layout.width.__getValue();
         const currentValue =
-          I18nManager.isRTL && axis === 'dx'
+          (I18nManager.isRTL && axis === 'dx') || gestureDirectionInverted
```

Comment:
My opinion:

1. File an issue on the right-edge back navigation not working
2. Leave it out of the scope of this PR
3. If you've tested what happens when `18nManager.isRTL == true`, and confirmed that it has no effect or a barely noticeable effect, leave this as is. Otherwise, I think it's best to assume the intent of the coder here, which I think is that `I18nManager.isRTL` should invert the direction of the inversion, so as to assume to amount of work the fixer of the aforementioned issue will have to do

Reply:
Okay, thanks. I've opened a new issue for this in #3127

Reply:
Thanks for filing that issue! Given that the behavior seems broken either way, I'd probably make it so `gestureDirectionInverted` inverts the result of `I18nManager.isRTL && axis === 'dx'` (so xor instead of or), so it's slightly less work for whoever ends up picking up #3127